src/ui/public/chrome/api/nav.js

```diff
@@ -26,24 +29,13 @@ export function initChromeNavApi(chrome, internals) {
     return internals.basePath || '';
   };
 
+  /**
+   *
+   * @param url {string} a relative url. ex: /app/kibana#/management
+   * @return {string} the relative url with the basePath prepended to it. ex: rkz/app/kibana#/management
+   */
   chrome.addBasePath = function (url) {
-    const isUrl = url && isString(url);
-    if (!isUrl) return url;
-
-    const parsed = parse(url, true, true);
-    if (!parsed.host && parsed.pathname) {
-      if (parsed.pathname[0] === '/') {
-        parsed.pathname = chrome.getBasePath() + parsed.pathname;
-      }
-    }
-
-    return format({
-      protocol: parsed.protocol,
-      host: parsed.host,
-      pathname: parsed.pathname,
-      query: parsed.query,
-      hash: parsed.hash,
-    });
+    return prependPath(url, chrome.getBasePath());
```

Comment:
I can modify prependPath to work on absolute paths as well as relative, but wanted to keep the functionality the same, for fear I will accidentally break something.